test/testdata/packager/export_for_test/foo/foo.test.rb

```diff
@@ -13,7 +13,6 @@ class Test::Opus::Foo::FooTest
 
   # util/__package.rb exposed via export_for_test, cannot access from here:
   Opus::Util::Nesting.nesting_method
-  #                   ^^^^^^^^^^^^^^ error: Method `nesting_method` does not exist on `T.class_of(Opus::Util::Nesting)`
```

Comment:
Is it ok that we're dropping this error?

Reply:
I think this worked before because we would re-create the tree leading up to `Opus::Util::Nesting::Public`, and so `Opus::Util::Nesting` would be a new symbol that existed with no methods. As parts of the Stripe codebase rely on naming the leading (but not explicitly exported) parts of a name, I'm not sure that we can keep this error, though we do have `package_private` to fall back on when this behavior is needed.